From a1150615f3c85993178e517b4601e04bd5b47125 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 8 Feb 2020 23:11:26 +0100
Subject: [PATCH 1/1] log: correct CONFIG_LOG_TEST prerequisites

An error

	undefined reference to `do_log_test'

occurs for CONFIG_CMD_LOG=y, CONFIG_LOG_TEST=y, CONGIG_UNIT_TEST=n

Make CONFIG_UNIT_TEST a prerequisite.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
Reviewed-by: Simon Glass <sjg@chromium.org>
---
 common/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/Kconfig b/common/Kconfig
index 16a67f5f76..5ad23ce2c4 100644
--- a/common/Kconfig
+++ b/common/Kconfig
@@ -782,7 +782,7 @@ config LOG_SYSLOG
 
 config LOG_TEST
 	bool "Provide a test for logging"
-	depends on LOG
+	depends on LOG && UNIT_TEST
 	default y if SANDBOX
 	help
 	  This enables a 'log test' command to test logging. It is normally
-- 
2.24.1

